.stylelintrc.yml

```diff
@@ -0,0 +1,6 @@
+extends:
```

Comment:
We can add our own file of stylelint rules, including comments in the rules to give context to the errors.

[Example](https://github.com/jorgeatgu/base/blob/master/.stylelintrc)